app/models/user.rb

```diff
@@ -4,6 +4,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_many :articles
+  has_many :likes, as: :likable
```

Comment:
I think `user` -> `likes` relationship should not be as `likeable`. Article -> like should be as likeable